src/AppBundle/Repository/EventWikiRepository.php

```diff
@@ -293,13 +292,9 @@ public function getPageviews(
 
         $pageviewsRepo = new PageviewsRepository();
         $recentDayCount = Event::AVAILABLE_METRICS['pages-improved-pageviews-avg'];
-
-        // The offset date is the start of the period over which pageviews should be averaged per day, up to today.
-        $offsetDate = (new DateTime('yesterday midnight'))->sub(new DateInterval('P'.$recentDayCount.'D'));
-        $start = $getDailyAverage && $start < $offsetDate ? $offsetDate : $start;
```

Comment:
This logic was moved to `PageviewsRepository::getAvgPageviewsPerArticle()` a while back and I forgot to remove it here. When `$getDailyAverage` is set then `$start` isn't used, so basically this code didn't do anything.